lazyflow/utility/helpers.py

```diff
@@ -250,7 +250,45 @@ def __iter__(self):
         retSlice = [(self.toSlice(src,True),self.toSlice(trgt,True),self.toSlice(mask)) \
                     for src,trgt,mask in retRoi]
         return retSlice.__iter__()
- 
+
+
+def get_default_axisordering(shape):
+    """Given a data shape, return the default axis ordering
+
+    For data types that do not support axistags, we assume a default axis
+    ordering, given the shape, and implicitly the number of dimensions.
+
+    Args:
+        shape (tuple): Shape of the data
+
+    Returns:
+        str: String, each position represents one axis.
+    """
+    axisorders = {
+        2: 'yx',
+        3: 'zyx',
+        4: 'zyxc',
+        5: 'tzyxc'
+    }
```

Comment:
Hi @stuarteberg,

thank you for looking at the PR.

I have used what I have found in the lazyflow code, so the default behaviour _now_ is unchanged.
At least we can control it from a single point now. Should we leave the PR open until we can have a discussion on that?

Or, would you say, that default axis ordering depends on the file type and should be handled more flexibly?